lib/charms/mongodb/v0/helpers.py

```diff
@@ -64,7 +66,7 @@ def get_create_user_cmd(config: MongoDBConfiguration, mongo_path="mongo") -> Lis
     ]
 
 
-def get_mongod_cmd(config: MongoDBConfiguration) -> str:
+def get_mongod_cmd(config: MongoDBConfiguration, mongo_files_dir="/etc/mongodb") -> str:
     """Construct the MongoDB startup command line.
 
     Returns:
```

Comment:
can you please add `--dbpath` arg with path to DATA_DIR? I want to have it passed via arg